data/sql_updates/create_candidates_view.sql

```diff
@@ -28,17 +29,25 @@ select
     max(dimoffice.office_state) as state,
     max(candprops.cand_nm) as name
 from dimcand
-    left join (select distinct on (cand_sk) cand_sk, election_yr, cand_status, ici_code from dimcandstatusici order by cand_sk, election_yr desc) csi_recent using (cand_sk)
-    left join dimcandoffice co on co.cand_sk = dimcand.cand_sk
+    left join (
+        select distinct on (cand_sk) cand_sk, election_yr, cand_status, ici_code
+            from dimcandstatusici
+            order by cand_sk, election_yr desc
+    ) csi_recent using (cand_sk)
+    left join dimcandoffice co using (cand_sk)
     inner join dimoffice using (office_sk)
     inner join dimparty using (party_sk)
     left join (
         select distinct on (cand_sk) cand_sk, cand_nm from dimcandproperties
             order by cand_sk, candproperties_sk desc
-    ) candprops on dimcand.cand_sk = candprops.cand_sk
+    ) candprops using (cand_sk)
+    left join dimcandproperties dcp using (cand_sk)
+    where dcp.election_yr >= :START_YEAR
```

Comment:
When comparing start year to an election year, make sure the beginning of the cycle is included